indico/modules/events/editing/client/js/management/editable_type/EditableTypeDashboard.jsx

```diff
@@ -9,6 +9,10 @@
 
 import anonymousTeamURL from 'indico-url:event_editing.api_anonymous_team';
 import enableEditingURL from 'indico-url:event_editing.api_editing_enabled';
+import emailMetadataURL from 'indico-url:event_editing.api_email_not_submitted_metadata';
+import emailPreviewURL from 'indico-url:event_editing.api_email_not_submitted_preview';
+import emailSendURL from 'indico-url:event_editing.api_email_not_submitted_send';
+import noEditablesNotSubmittedURL from 'indico-url:event_editing.api_no_editables_not_submitted';
```

Comment:
This looks like a double negative and my brain fails to parse it :sweat_smile:
maybe `numberEditablesNotSubmitted` or `countEditablesNotSubmitted`? Or even something like `ContribsWithNoEditables` since it actually counts contributions and not editables? What do you think?